packages/opentelemetry-exporter-prometheus/src/prometheus.ts

```diff
@@ -160,7 +160,7 @@ export class PrometheusExporter implements MetricExporter {
   }
 
   private _getLabelValues(keys: string[], labels: api.Labels) {
-    const labelValues: labelValues = {};
+    LabelValues<string> = {};
```

Comment:
This overwrites the variable from `prom-client` which I'm sure is not what was intended. Did you mean to do this?
```suggestion
    const labelValues: LabelValues<string> = {};
```

Reply:
This was by mistake, i wanted to add commit to other PR.